### PR TITLE
Mongodb setup

### PR DESCRIPTION
**Pull request by hdJerry**
- Configured mondoDB
- user models
- user models for favourite and read

### CODE REVIEW

File: server/activities.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const user_activitie = {
```
Comment (kenslachtajr) [inline server/activities.js]: typo - "activity"

File: server/models/users/users-read.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  courses: {
+    type: Array,
+    default: [],
+  },
+  tutorials: {
+    type: Array,
+    default: [],
+  },
+  hackatons: {
```
Comment (kenslachtajr) [inline server/models/users/users-read.js]: "hackathons"

File: server/activities.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+  favourites: 'favourite',
```
Comment (kenslachtajr) [inline server/activities.js]: favourites: 'favourites',

File: server/models/users/users-favourite.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  courses: {
+    type: Array,
+    default: [],
+  },
+  tutorials: {
+    type: Array,
+    default: [],
+  },
+  hackatons: {
```
Comment (kenslachtajr) [inline server/models/users/users-favourite.js]: "hackathons"

File: server/models/users/users-favourite.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    type: Array,
+    default: [],
+  },
+});
+
+const dbUSERSFAVOURITE = mongoose.model('usersfavourites', USERSFAVOURITE);
```
Comment (kenslachtajr) [inline server/models/users/users-favourite.js]: "dbUSERSFAVOURITES"?
"USERSFAVOURITES"?

**Review by kenslachtajr — CHANGES_REQUESTED** [review]

File: server/activities.js (hunk 1 above)
Comment (hdJerry) [reply server/activities.js]: it is missing an `s`. sorry

**Review by kenslachtajr — APPROVED** [review]
LGTM

**Pull request merged**